zerver/lib/markdown/merge_api_changelogs.py

```diff
@@ -0,0 +1,140 @@
+import glob
+import os
+import re
+from pathlib import Path
```

Comment:
This isn't the right place for this; this code has nothing to do with the Markdown processor. 

I think `zerver/openapi/merge_api_changelogs.py` would be appropriate.